content/en/docs/concepts/configuration/organize-cluster-access-kubeconfig.md

```diff
@@ -17,6 +17,11 @@ a *kubeconfig file*. This is a generic way of referring to configuration files.
 It does not mean that there is a file named `kubeconfig`.
 {{< /note >}}
 
+{{< warning >}}
+Only use kubeconfig files from trusted sources. Using a specially-crafted kubeconfig file could result in malicious code execution or file exposure. 
```

Comment:
Hi @gracenng .
The warning callout looks okay.
What is a "specially-crafted" kubeconfig file?
I *think* the second sentence is referring to possible exposure to malware?

Reply:
Yes. I'm trying to say that kubeconfig file crafted with malicious intent can contain malware, but not all kubeconfig files are dangerous by nature. Do let me know if you have other questions

Reply:
To help amplify what Grace is saying, there are several ways Kubeconfig files can be misused:
* at least two features cause shell scripts embedded in the kubeconfig file to be executed when you use that context
* Several more features send the contents of a specified file to the specified webserver, which can be misused to send the contents of a sensitive file owned by the user to a server controlled by the attacker

These aren't vulnerabilities, but they are features that are not very well known and people do not generally realize the importance of trusting the source of kubeconfig files before using them.

Reply:
Thanks @gracenng .
/approve